src/pages/LogInWithShortLivedAuthTokenPage.js

```diff
@@ -62,7 +74,12 @@ class LogInWithShortLivedAuthTokenPage extends Component {
     }
 
     render() {
-        if (this.props.isTokenValid) {
+        // TODO: Not sure if we should redirect the user to the login page or just show the expiration screen or offline message
```

Comment:
Same comment as above about TODOs (unless you are meaning to clean up this TODO in this PR)